.github/workflows/workflow_canceller.yml

```diff
@@ -0,0 +1,19 @@
+name: Automatic Workflow Canceller
+
+on:
```

Comment:
We can add a top-level comment on when this action is going to get triggered.

Reply:
Added some details around this.